### PR TITLE
https://issues.redhat.com/browse/ACM-12313

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: governance/policy_overview.adoc [hunk 1]
```diff
@@ -8,8 +8,51 @@ You are responsible for ensuring that the managed cloud environment meets intern
...
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-policy-role
+  placementRef:
+    name: placement-policy-role 
+    kind: Placement 
+    apiGroup: cluster.open-cluster-management.io
```
Comment (gparvin) [inline governance/policy_overview.adoc]:
```suggestion
placementRef:
  name: placement-policy-role 
  kind: Placement 
  apiGroup: cluster.open-cluster-management.io
```
Remove two spaces from the beginning of each of these lines

File: governance/policy_overview.adoc [hunk 2]
```diff
@@ -8,8 +8,51 @@ You are responsible for ensuring that the managed cloud environment meets intern
...
+      subjects:
+      - name: policy-role
+        kind: Policy 
+        apiGroup: policy.open-cluster-management.io
```
Comment (gparvin) [inline governance/policy_overview.adoc]:
```suggestion
subjects:
- name: policy-role
  kind: Policy 
  apiGroup: policy.open-cluster-management.io
```
Remove some spaces here too so it looks as shown.

**Review by gparvin — APPROVED** [review]
This looks good to me now.  Thanks

File: governance/policy_overview.adoc [hunk 3]
```diff
@@ -8,8 +8,51 @@ You are responsible for ensuring that the managed cloud environment meets intern
...
+  name: placement-policy-role 
+  kind: Placement 
+  apiGroup: cluster.open-cluster-management.io
+subjects:
+- name: policy-role
+  kind: Policy 
+  apiGroup: policy.open-cluster-management.io
+----
++
+*Note:* If you use the previous sample, make sure you update the name of the `Placement` resource in the `placementRef` section to match your placement name. You must also update the name of the policy in the `subjects` section to match your policy name. Use the `oc apply -f resource.yaml -n namespace` command to apply both the `Placement` and the `Placementbinding` resources. Make sure the policy, placement and placement binding are all created in the same namespace.
```
Comment (oafischer) [inline governance/policy_overview.adoc]: Wondering if it's better to break out some of this, I think the references to the samples can be callouts. I would also add the command to it's own paragraph.

**Review by oafischer — APPROVED** [review]
One suggestion, but feel free to merge.

**Comment by dockerymick** [comment]
@gparvin @oafischer Thanks to you both for your reviews and approvals. Merging this PR

**Pull request merged**